packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/sagas.ts

```diff
@@ -1069,7 +1069,6 @@ export default ({
         // REMOVE THIS WHEN BACKEND CAN HANDLE PENDING 'FUNDS' ORDERS
         // 👇--------------------------------------------------------
         case 'BANK_ACCOUNT':
-          // case 'USER_CARD': commenting out until we figure out SDD issues
```

Comment:
shouldnt this be added back? it was there for SDD

Reply:
@schnogz I'll check since we have BE ready now